src/test/java/seedu/address/stubs/delete/ModelManagerStub.java

```diff
@@ -167,4 +167,9 @@ public ObservableList<AttendanceEvent> getAttendanceEventList() {
     public List<Person> getStudentsByAttendance(String eventName, boolean isPresent) {
         return Collections.emptyList();
     }
+
+    @Override
+    public void deleteAttendanceEvent(AttendanceEvent event) {
```

Comment:
Hmm should the class still be overridden if you planning to make most of the inherited methods redundant?

Reply:
you have to override the methods on abstract model in the stub to avoid compilation errors.